truss/templates/base.Dockerfile.jinja

```diff
@@ -2,9 +2,10 @@ ARG PYVERSION={{config.python_version}}
 FROM {{base_image_name_and_tag}}
 
 {% block fail_fast %}
+ARG PYVERSIONCHECK="import sys; sys.exit(0) if sys.version_info.major == 3 and sys.version_info.minor >=8 and sys.version_info.minor <=11 else sys.exit(1)"
 RUN grep -w 'ID=debian\|ID_LIKE=debian' /etc/os-release || { echo "ERROR: Supplied base image is not a debian image"; exit 1; }
 RUN pythonVersion=$(echo $(command -v python >/dev/null 2>&1 && python --version || python3 --version) | cut -d" " -f2 | cut -d"." -f1,2) \
-    && python -c "import sys; sys.exit(0) if sys.version_info.major == 3 and sys.version_info.minor >=8 and sys.version_info.minor <=11 else sys.exit(1)" \
+    && (python -c "$PYVERSIONCHECK" || python3 -c "$PYVERSIONCHECK") \
```

Comment:
This was throwing an error on both the default GPU base image and my custom image, as both only have `python3` available, not `python`. I also noticed that the entrypoint below specifies `python3`, so either that needs to also be dynamic or this serves no real purpose.

Reply:
Thanks for the comment. I think @joostinyi put up a fix in a separate PR